SparkPost/matchbox
# UX-371 Replace all images with cropped versions

### What Changed
- Replaces all images in `matchbox-media` with cropped versions


### How To Test or Verify
- Run storybook and visit http://localhost:9001/?path=/story/media-images--all-images to verify images

### PR Checklist
